notebooks/tfx_pipelines/cicd/solutions/tfx_cicd_vertex.ipynb

```diff
@@ -0,0 +1,313 @@
+{
```

Comment:
<p>Suggestion: to help frame learner expectations on runtime, consider adding something like:</p><p> </p><p>"Note: building and deploying the container below is expected to take 10-15 min."</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/asl-ml-immersion/pull/145/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='15'/>

Reply:
Similarly, consider adding a note during the first manual trigger, something like:

"Note: this step will build, deploy, and start a pipeline run. The full pipeline run with take about 1 hour 10 min; please continue on to the next exercise."

Reply:
<p>Done!</p>